pkg/pipelines/utils.go

```diff
@@ -86,8 +86,14 @@ func pushRepository(o *BootstrapOptions, remote string, e executor) error {
 	if out, err := e.execute(o.OutputPath, "git", "branch", "-m", "main"); err != nil {
 		return fmt.Errorf("failed to switch to branch 'main' in repository in %q %q: %s", o.OutputPath, string(out), err)
 	}
-	if out, err := e.execute(o.OutputPath, "git", "remote", "add", "origin", remote); err != nil {
-		return fmt.Errorf("failed add remote 'origin' %q to repository in %q %q: %s", remote, o.OutputPath, string(out), err)
+	if _, err := e.execute(o.OutputPath, "git", "remote", "show", "origin"); err != nil {
+		if out, err := e.execute(o.OutputPath, "git", "remote", "add", "origin", remote); err != nil {
```

Comment:
we can probably just have an error check here? 
if `err contains "remote origin already exists"` we can call `git remote set-url`, if the error is something else just return the error.
What do you think?